src/main/java/core/basesyntax/BuilderTestApp.java

```diff
@@ -1,7 +1,15 @@
 package core.basesyntax;
 
 public class BuilderTestApp {
-  public static void main(String[] args) {
-      // Remove this comment and test your Builder implementation here...
-  }
+
+    public static void main(String[] args) {
+        Plane plane = new Plane.PlaneBuilder()
+                .setModel("Airbus 383")
+                .setFlyDistance(1500)
+                .setPassengersCapacity(118)
+                .setIsIntercontinental(true)
+                .setHasBusinessClass(false)
+                .build();
+        plane.toString();
```

Comment:
again, how can you see the result of this statement? let's print it out?

Reply:
I see I forgot to add System.out.pringln  - sorry